suite-native/accounts/src/components/AccountBalance.tsx

```diff
@@ -33,7 +33,7 @@ export const AccountBalance = ({ accountKey, accountName }: AccountBalanceProps)
             <Text style={applyStyle(accountNameStyle)}>{accountName}</Text>
             <Box flexDirection="row" alignItems="center">
                 <Box style={applyStyle(cryptoIconStyle)}>
-                    <CryptoIcon size="large" name="btc" />
+                    <CryptoIcon size="large" name={account.symbol as CryptoIconName} />
```

Comment:
Did you tried to rebase and remove this typecasting? I think @bouchja1 improved types for CryptoIcons recently.

Reply:
removed